src/main/java/games/strategy/engine/lobby/server/login/RsaAuthenticator.java

```diff
@@ -171,8 +174,23 @@ static String hashPasswordWithSalt(final String password) {
     return Collections.singletonMap(ENCRYPTED_PASSWORD_KEY, encryptPassword(challenge.get(RSA_PUBLIC_KEY), password));
   }
 
+  /**
+   * This method exists only to help support test.
+   * @deprecated Avoid calling this method, instead we should rework how this class is structured
+   */
+  @Deprecated
   @VisibleForTesting
   static void invalidateAll() {
     rsaKeyMap.invalidateAll();
   }
+
+  /**
+   * This method exists only to help support test.
```

Comment:
Just a suggestion for a future PR because it's probably out of scope for this one...

Maybe let the key pair cache be owned by `LobbyLoginValidator` and pass it in to `RsaAuthenticator` via constructor injection.  That would obviously change how `RsaAuthenticator` is used because now an instance must be created instead of simply calling a collection of static methods.  But that might be worth it to avoid the global state of the `rsaKeyMap` class field.